x-pack/legacy/plugins/reporting/server/routes/lib/job_response_handler.ts

```diff
@@ -67,3 +68,34 @@ export function jobResponseHandlerFactory(
     });
   };
 }
+
+export function deleteJobResponseHandlerFactory(
```

Comment:
This is essentially duplicated as the downloadJobResponseHanderFactory code, but it uses async/await